### ISSUE TITLE
ERROR Cannot destructure property 'tag' of 'undefined' as it is undefined.

### ISSUE THREAD

**Issue by kaikaibenkai** [nuxt/nuxt#19585]
### Environment

- Operating System: `Windows_NT`
- Node Version:     `v18.15.0`
- Nuxt Version:     `3.2.3`
- Nitro Version:    `2.2.3`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

Download [Minimal Starter](https://codeload.github.com/nuxt/starter/tar.gz/refs/heads/v3), run `yarn` to install dependencies, and then errors occurred.

### Describe the bug

Failed to generate `.nuxt` in `postinstall` hook.

### Logs

```shell-script
$ yarn
yarn install v1.22.19
info No lockfile found.
[1/4] Resolving packages...
warning nuxt > vue > @vue/compiler-sfc > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
warning nuxt > @nuxt/vite-builder > cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
[2/4] Fetching packages...
warning vscode-languageclient@7.0.0: The engine "vscode" appears to be invalid.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
$ nuxt prepare
Nuxi 3.2.3                                                                                                     23:33:04

 ERROR  Cannot destructure property 'tag' of 'undefined' as it is undefined.                                   23:33:07  

  at tag:normalise (/D:/site/fanbook-bot-tools/my-app/node_modules/unhead/dist/index.mjs:209:35)
  at /D:/site/fanbook-bot-tools/my-app/node_modules/hookable/dist/index.mjs:47:66
  at Object.run (/D:/site/fanbook-bot-tools/my-app/node_modules/hookable/dist/index.mjs:41:43)
  at /D:/site/fanbook-bot-tools/my-app/node_modules/hookable/dist/index.mjs:47:56
  at async Object.resolveTags (/D:/site/fanbook-bot-tools/my-app/node_modules/unhead/dist/index.mjs:933:11)
  at async renderSSRHead (/D:/site/fanbook-bot-tools/my-app/node_modules/@unhead/ssr/dist/index.mjs:77:23)
  at async initNitro (/D:/site/fanbook-bot-tools/my-app/node_modules/nuxt/dist/index.mjs:1915:21)
  at async initNuxt (/D:/site/fanbook-bot-tools/my-app/node_modules/nuxt/dist/index.mjs:2262:3)
  at async loadNuxt (/D:/site/fanbook-bot-tools/my-app/node_modules/nuxt/dist/index.mjs:2289:5)
  at async loadNuxt (/D:/site/fanbook-bot-tools/my-app/node_modules/@nuxt/kit/dist/index.mjs:522:19)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by noook** [comment]
It seems that some hook parameter are undefined, I had the same problem a few minutes before with other hooks where the parameters are undefined

```ts
nuxt.hook("imports:sources", (sources) => {
  // Cannot read property `find` of undefined (sources)
  const routerImports = sources.find((s) => s.from === "#app" && s.imports.includes("onBeforeRouteLeave"));
  if (routerImports) {
    routerImports.from = "vue-router";
  }
});
```

and

```ts
nuxt.hook("imports:extend", (imports) => {
  // Cannot read property `push` of undefined (imports)
  imports.push(
    { name: "definePageMeta", as: "definePageMeta", from: resolve(runtimeDir, "composables") },
    { name: "useLink", as: "useLink", from: "vue-router" }
  );
});
```

**Comment by noook** [comment]
I wonder if it has something to do with the latest release of `hookable` https://github.com/unjs/hookable/releases/tag/v5.5.0

With a bit of debugging the `callHook` function seems to be the one that causes the undefined parameter. Not a 100% sure but I'll keep digging

This PR seems to be the fix for it https://github.com/unjs/hookable/pull/71

**Comment by plcdnl** [comment]
as a temporary fix you can force to use hookable 5.4.2 in package.json
